src/wallet/coincontrol.h

```diff
@@ -24,6 +24,33 @@ const int DEFAULT_MAX_DEPTH = 9999999;
 //! Default for -avoidpartialspends
 static constexpr bool DEFAULT_AVOIDPARTIALSPENDS = false;
 
+class PreselectedInput
```

Comment:
in 59a7c7416313693a3c8219363d9ffee1e2b86d4d

Naming is hard. We already have struct `PreSelectedInputs` which is quite confusing as it refers to different data.

`PreSelectedInputs` is basically `set<COutput>`. It seems we should find a better name for `PreSelectedInputs`. Maybe rename it to `PreSelectedCoins`?

It'd be nice to have consistent terms to refer to different data.
How about we use `input` when we talk about tx.in and related info.
And we use `coin` when its enriched data in context of coin selection.

Perhaps @murchandamus have thought about terminology.

Reply:
:shrug: